src/main/java/net/mindengine/galen/specs/reader/page/PageSpecReader.java

```diff
@@ -114,6 +116,9 @@ public PageSpec read(InputStream inputStream, String fileLocation, String contex
                 lineNumber++;
             }
         }
+        catch (WebDriverException webDriverException) {
```

Comment:
otherwise it would use the general Exception catch block and throw a SyntaxException